brands/shop/houseware.json

```
@@ -175,6 +175,18 @@
         "shop": "houseware"
       }
     },
+    {
+      "displayName": "Prince Hypermart",
+      "id": "princehypermart-52d9d6",
+      "locationSet": {"include": ["ph", "phl"]},
```

Just `ph` is ok here.. (`phl` is the same thing, but the 3 letter code)
```suggestion
      "locationSet": {"include": ["ph"]},
```

---

updated! please review thanks!